src/routes/(protected)/onboarding/+page.svelte

```diff
@@ -11,6 +11,7 @@
 
 	export let currentColorScheme = $page.data.session?.user?.settings.colorScheme;
 	export let currentLocale = $page.data.session?.user?.settings.localeId;
+	export let fieldErrors = $page.form?.fieldErrors;
```

Comment:
Update to use ActionData like https://kit.svelte.dev/docs/form-actions#anatomy-of-an-action-validation-errors
```ts
import type { ActionData } from './$types';
export let form: ActionData;
```

`form` will have anything we return in the form action (or the object we return in fail)